src/many-ledger/src/module.rs

```diff
@@ -821,6 +837,45 @@ impl idstore::IdStoreModuleBackend for LedgerModuleImpl {
     }
 }
 
+impl DataModuleBackend for LedgerModuleImpl {
```

Comment:
First, this is not a "migration", this is just an additional module, modules do not migrate. Second, it would not be possible to put it in a separate module because `DataModuleBackend` is defined externally, so the trait impl needs to be in the same module as the `LedgerModuleImpl` definition.

Reply:
Sorry, you could do a `module/data.rs` or the like and place the data-related impl for the LedgerModuleImpl in it.

I'm doing that in `kvstore` to lighten `module.rs`, e.g., https://github.com/liftedinit/many-framework/blob/2b8bde732618f9342f2f4aeb367d69a250057862/src/many-kvstore/src/module/account.rs#L119